### ISSUE TITLE
How to use Orca2 language model

### ISSUE THREAD

**Issue by zydjohnHotmail** [Troyanovsky/Local-LLM-Comparison-Colab-UI#13]
Hello:
I am new to Orca2 language model, I have no idea on how to use this, but I have a real world example, so I want to know how to use t this language model to split the words correctly for those OCR text returned by Florence2 language model.
See the screenshot to know the OCR test image.
The OCR returned by Florence2 language model looks like this:
This is a lot of 12 point text to test theocr code and see if it works on all ctypesof file format.The quick brown dog jumped over thelazy fox. The quick brown dog jumpedover the lazy fox.Thequick brown dogjumped over the lazyfox. Thequickbrown dog jumpedoverthe lazy fox
Basically,  the OCR results are quite OK, but not exactly.  I want to use Orca2 language model to correctly split the following strings:
theocr; jumpedover; dogjumped; lazyfox.  Thequickbrown; jumpedoverthe.  Even in the original image, there are clearly some spaces, but the OCR results did NOT have any space at all.
Please show me some code how to use this language to correctly split those words.
![testocr](https://github.com/user-attachments/assets/e3bfea51-9840-4c34-9ba8-fc2b5148cb9d)

**Comment by Troyanovsky** [comment]
You can try out the model with web UI from this repo here: [Orca 2 13B](https://github.com/Troyanovsky/Local-LLM-Comparison-Colab-UI/blob/main/Orca_2_13B_GGUF.ipynb), [Orca 2 7B](https://github.com/Troyanovsky/Local-LLM-Comparison-Colab-UI/blob/main/Orca_2_7B_GGUF.ipynb). Hit "Open In Colab", then follow the instructions provided in the notebook.

Then you can run the model locally with [llama-cpp-python](https://github.com/abetlen/llama-cpp-python) or [ollama](https://ollama.com/library/orca2) on your local machine. Their repo provides guides on how to run them.

Simply input a prompt like:
```
Some words are incorrectly combined, please split them: This is a lot of 12 point text to test theocr code and see if it works on all ctypesof file format.The quick brown dog jumped over thelazy fox. The quick brown dog jumpedover the lazy fox.Thequick brown dogjumped over the lazyfox. Thequickbrown dog jumpedoverthe lazy fox
```
